### PR TITLE
feat: add google analytics tracking

### PR DESCRIPTION
**Pull request by vinteo**
## Description

Add Google Analytics tracking for production Adslot UI website

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: www/containers/props.json [hunk 1]
```diff
@@ -1863,6 +1863,17 @@
       "displayName": "ImageCropper",
       "methods": [],
       "props": {
+        "title": {
```
Comment (vinteo) [inline www/containers/props.json:1866]: not sure why this was added automagically?

File: www/containers/GoogleAnalytics.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+
+const GA_TRACKING_ID = 'UA-18273448-24';
+
+class GoogleAnalytics extends React.Component {
+  componentWillUpdate({ location, history }) {
```
Comment (veegandhi) [inline www/containers/GoogleAnalytics.jsx:7]: this one is an unsafe life cycle. You can probably just `useEffect` on location and history

File: www/containers/props.json (hunk 1 above)
Comment (veegandhi) [reply www/containers/props.json:1866]: I think the it was previously not committed with the feature

File: www/index.html [hunk 3]
```diff
@@ -1,13 +1,24 @@
-<!doctype html>
+<!DOCTYPE html>
```
Comment (veegandhi) [inline www/index.html:1]: I know it's slightly confusing but this html is for dev bundling purposes so we don't load external dependency. For production we use the `index.html` in `src`. So this code will belong there.

Comment (vinteo) [reply www/index.html:1]: VSCode formatted the HTML, no changes in this file now besides indentation

**Comment by pphminions** [comment]
merging 🚀

**Pull request merged**